---
macd_sell_signal/macd[] macd_array